src/track/serato/markers.cpp

```diff
@@ -699,9 +713,13 @@ void SeratoMarkers::setCues(const QList<CueInfo>& cueInfos) {
                     static_cast<int>(*cueInfo.getStartPositionMillis()),
                     true,
                     static_cast<int>(*cueInfo.getEndPositionMillis()),
-                    *cueInfo.getColor(),
+                    // TODO: In Serato, saved loops always have a fixed color.
+                    // We *could* export the actual color here if we also
+                    // import the blue-ish default color in the code above, but
+                    // it will not be used by Serato.
+                    SeratoTags::kFixedLoopColor,
                     static_cast<int>(SeratoMarkersEntry::TypeId::Loop),
-                    false);
+                    cueInfo.flags().testFlag(CueFlag::Locked));
```

Comment:
How about adding and isLocked() convenience method to CueInfo to make the code more readable? But only for reading, while setting the flags must still be done all at once.